src/globus_cli/commands/gcs/endpoint/_common.py

```diff
@@ -0,0 +1,25 @@
+from globus_cli.termio import Field, formatters
+
+# https://docs.globus.org/globus-connect-server/v5.4/api/schemas/Endpoint_1_2_0_schema/
+GCS_ENDPOINT_FIELDS = [
+    Field("Endpoint ID", "id"),
+    Field("Display Name", "display_name"),
+    Field("Allow UDT", "allow_udt", formatter=formatters.FuzzyBool),
+    Field("Contact Email", "contact_email"),
+    Field("Contact Info", "contact_info"),
+    Field("Department", "department"),
+    Field("Description", "description"),
+    Field("Earliest Last Access", "earliest_last_access", formatter=formatters.Date),
+    Field("GCS Manager URL", "gcs_manager_url"),
+    Field("GridFTP Control Channel Port", "gridftp_control_channel_port"),
+    Field("Info Link", "info_link"),
+    Field("Keywords", "keywords"),
```

Comment:
I think that GCS returns keywords as a list. I don't fully remember how we made formatting work. Do we need to specify an array formatter for this?

Reply:
It currently renders like this:

<img width="771" alt="image" src="https://github.com/globus/globus-cli/assets/113056046/802f0ea1-e7f3-4afe-bdfb-bf7c1134600e">

---

With `Field("Keywords", "keywords", formatter=formatters.Array)` it renders like this

<img width="763" alt="image" src="https://github.com/globus/globus-cli/assets/113056046/26ce14e5-bf3e-4ddc-8999-12472b30c51a">

Reply:
Updating rendering to `formatters.Array`. New Rev